test/tools/jsontests/vm.cpp

```diff
@@ -480,114 +474,55 @@ json_spirit::mValue doVMTestsNoLog(json_spirit::mValue const& _input, bool _fill
 
 } } // namespace close
 
-BOOST_AUTO_TEST_SUITE(VMTests)
-
-BOOST_AUTO_TEST_CASE(vmtests)
-{
-	dev::test::executeTests("vmtests", "/VMTests", "/VMTestsFiller", dev::test::doVMTests);
-}
-
-BOOST_AUTO_TEST_CASE(vmArithmeticTest)
-{
-	dev::test::executeTests("vmArithmeticTest", "/VMTests", "/VMTestsFiller", dev::test::doVMTests);
-}
-
-BOOST_AUTO_TEST_CASE(vmBitwiseLogicOperationTest)
-{
-	dev::test::executeTests("vmBitwiseLogicOperationTest", "/VMTests", "/VMTestsFiller", dev::test::doVMTests);
-}
-
-BOOST_AUTO_TEST_CASE(vmSha3Test)
-{
-	dev::test::executeTests("vmSha3Test", "/VMTests", "/VMTestsFiller", dev::test::doVMTests);
-}
 
-BOOST_AUTO_TEST_CASE(vmEnvironmentalInfoTest)
+class vmtestfixture
```

Comment:
I have no idea, why out of nowhere you named this class this way.

Reply:
following the same scheme as for the other tests.

Reply:
Like `ConstantinopleTransitionTestFixture` or `TestOutputFixture`?

Reply:
like transactiontestfixture, generaltestfixture

Reply:
Let's do CamelCase in class names.